docs/snippets/ov_preprocessing_migration.cpp

```diff
@@ -5,8 +5,18 @@
 #include <openvino/opsets/opset8.hpp>
 #include <openvino/core/preprocess/pre_post_process.hpp>
 
+#ifndef IN_OV_COMPONENT
+#    define IN_OV_COMPONENT
+#    define WAS_OV_LIBRARY_DEFINED
+#endif
```

Comment:
should we implement macro `OPENVINO_SUPPRESS_CPP_WARNING_START` / `OPENVINO_SUPPRESS_CPP_WARNING_END` ?